scripts/new.lua

```diff
@@ -8,160 +8,284 @@ import("devel.git")
 import("utils.archive")
 
 local options = {
-    {nil, "repo", "v", nil, "Set repository name.",
-                            "e.g. ",
-                            "  - github:xmake-io/xmake",
-                            "  - brew:zlib"}
+    {
+        nil,
+        "repo",
+        "v",
+        nil,
+        "Set repository name.",
+        "e.g. ",
+        "  - github:xmake-io/xmake",
+        "  - gitlab:xmake-io/xmake",
+    },
```

Comment:
Do not change the indentation format of the code, please ensure minimal code changes.